js/Source/Delegators/Delegator.InputMirror.js

```diff
@@ -34,9 +39,17 @@ Delegator.register(['change','keyup'], {
         var targetElement = element.getElement(target.selector);
         if (!targetElement) api.warn('Unable to find element for inputMirror selector: '+target.selector);
         if (targetElement && targetElement != element){
-          targetElement.set(target.property || 'value', element.get('value'));
+          var value = element.get('value');
+          if (api.get('stripHTML')){
+            value = value.replace(/</g, '&#60;').replace(/>/g, '&#62;').replace(/\//g, '&#47;');
+          }
```

Comment:
we might want to do strip scripts? http://mootools.net/core/docs/1.5.2/Types/String#String:stripScripts